resources/js/notification_order_status.js

```diff
@@ -6,17 +6,29 @@ window.Echo = new Echo({
     encrypted: true
 });
 
+var countAudio = 0;
+$(document).mousemove(function(){
+    $(document).click(function(){
+            if(countAudio == 0){
+                document.getElementById('audio').play();
+                countAudio+=1;
+            }
+    })
+})
```

Comment:
```suggestion
$(document).mousemove(function() {
    $(document).click(function() {
        if (countAudio == 0) {
            document.getElementById('audio').play();
            countAudio+=1;
        }
    })
})
```
indent 4 spaces, fix tương tự các chỗ khác